README.md

```diff
@@ -27,6 +27,8 @@ Usage of goma:
     	Basic Auth username
   -v int
     	Verbosity level [0,1,2,3] (default 1)
+  -henrique string
```

Comment:
This doc says the param is `-henrique`, but later it's defined as `Henrique`. Let's make it one way or another.